src/main/java/org/mifos/connector/channel/zeebe/ZeebeWorkers.java

```diff
@@ -59,6 +61,10 @@ public void setupWorkers() {
         workerSendPayeeFailureToChannel();
         workerInvokeAcknowledgementWorkflows();
         zeebeConsistencyWorker();
+<<<<<<< Updated upstream
```

Comment:
please remove these conflict markers. Also make sure the branch is rebased properly

Reply:
yes